### PR TITLE
Grpc Support

### PR DESCRIPTION
**Pull request by heitorPB**
Solves #DO-1472

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -188,3 +195,133 @@ func Ping(address string, config *Config) (bool, time.Duration) {
...
+		Formatter:      formatter,
+		VerbosityLevel: 0, // Zero is the default. Increasing this value
+		                   // increases the verbosity of the output of
+		                   // the RPC invocation, adding request
+		                   // destination, response header, etc. which
+		                   // messes up parsing the "BODY".
+	}
+
+	var headers []string
+	err = grpcurl.InvokeRPC(ctx, ds, client, service, headers, handler, rf.Next)
```
Comment (heitorPB) [inline client/client.go:321]: Something is missing here. Gatus is erroring out in our staging gateway with `server does not support the reflection API` but with grpcurl I can make the same request and even `list` and see the `grpc.reflection.v1alpha.ServerReflection`...

What could it be?

Comment (heitorPB) [reply client/client.go:321]: It works for other servers though.

Comment (heitorPB) [reply client/client.go:321]: Or maybe the issue is not on this code?

Comment (0xAleksaOpacic) [reply client/client.go:321]: Can you share which server is causing the issue

**Review by 0xAleksaOpacic — APPROVED** [review]
LGTM

**Pull request merged**